install.sh

```diff
@@ -37,8 +37,8 @@ apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docke
 ## By default, the Docker daemon requires root privileges to run. To use Docker without sudo, add your user to the Docker group
 usermod -aG docker $DDB_USER
 
-## Change permissions of the Docker socket
-chmod 666 /var/run/docker.sock
+## Activate the changes to groups
+newgrp docker
```

Comment:
Not needed anymore if I remember correctly : newgrp blocks the session, but the only place where this is needed is on the ddb installer, which already uses su -c to install